test/e2e/e2e.go

```diff
@@ -59,7 +60,9 @@ func SetupServingNamespaceforSecurityTesting(t *testing.T) *test.Clients {
 
 // SetupWithNamespace creates the client objects needed in the e2e tests under the specified namespace.
 func SetupWithNamespace(t *testing.T, namespace string) *test.Clients {
+	t.Helper()
 	pkgTest.SetupLoggingFlags()
+	t.Cleanup(logstream.Start(t))
```

Comment:
Is this guaranteed to be called from all e2e tests?

Reply:
There are no guarantees, but I generally tried to look for it whenever I was removing it.